src/BenchmarkDotNet/Loggers/AsyncProcessOutputReader.cs

```diff
@@ -89,13 +95,21 @@ private void Detach()
         private void ProcessOnOutputDataReceived(object sender, DataReceivedEventArgs e)
         {
             if (!string.IsNullOrEmpty(e.Data))
+            {
                 output.Enqueue(e.Data);
+                if (logOutput)
+                    logger.WriteLineInfo(e.Data);
+            }
         }
 
         private void ProcessOnErrorDataReceived(object sender, DataReceivedEventArgs e)
         {
             if (!string.IsNullOrEmpty(e.Data))
+            {
                 error.Enqueue(e.Data);
+                if (logOutput)
+                    logger.WriteLineInfo(e.Data);
```

Comment:
for error we can use `WriteLineError` and get red color:

```suggestion
                    logger.WriteLineError(e.Data);
```